tests/Type/WebMozartAssert/data/data.php

```diff
@@ -180,6 +180,14 @@ public function doFoo($a, $b, array $c, iterable $d, $e, $f, $g, $h, $i, $j, $k,
 
 		Assert::interfaceExists($ag);
 		\PHPStan\Testing\assertType('class-string', $ag);
+
+		/** @var int[] $at */
+		$at = doFoo();
+		Assert::count($at, 1);
+		$au = array_pop($at);
+		$av = array_pop($at);
+		\PHPStan\Testing\assertType('int', $au);
+		\PHPStan\Testing\assertType('int|null', $av);
```

Comment:
I'm wondering if the type should rather be `null` here.

Reply:
PHPStan doesn't report the exact count in array.